README.md

```diff
@@ -50,6 +50,22 @@ have dependencies on other algebras which must be implemented.
     - Two promises are equivalent when they yield equivalent values.
     - Two functions are equivalent if they yield equivalent outputs for equivalent inputs.
 
+## Type signature notation
+
+The type signature notation used in this document is described [here](https://sanctuary.js.org/#types). In summary:
+
+1. `->` (arrow) expresses a function's explicit parameter and return value types. `String` -> `[String]` -> `[String]` is a function which takes a `String` and an `[String]` and returns an `[String]`.
+2. `~>` (squiggly arrow) expresses a function's implicit parameter type. When a function is a property of a type, it is called a method. All methods have an implicit parameter type - the type of which they are a property of.
+3. `=>` (fat arrow) expresses constraints on type variables. For example, `a -> b` is a function which takes an argument and returns a result. The argument can be any type. The result can be any type (including the same type as the argument). All this applies to the function described by `Functor f => f a -> f b` but we now require the type `f` to satisfy the Functor specification.
+
+i.e.
+
+```
+sequence :: Apply f, Traversable t => t (f a) ~> (b -> f b) -> f (t a)
+'------'    '--------------------'    '-----'    '-------------------'
+   '- method name     '- constraints     '- method target  '- method arguments
```

Comment:
We now require `traverse` rather than `sequence`. It would be good to update this. Also, the section labelled "method arguments" is actually argument type and return type.